src/views/Home/index.test.tsx

```diff
@@ -120,7 +120,7 @@ describe('The home page', () => {
 
         setImmediate(() => {
           component.update();
-          expect(component.find(ActionBanner).length).toEqual(4);
+          expect(component.find(ActionBanner).length).toEqual(3);
```

Comment:
Why did this expectation change? Was the test written incorrectly before?

Reply:
You called out a thing I forgot to investigate more thoroughly. Turns out when I added biz case id to intake, I didn't add it to the initialSystemIntakeForm, which is used in many tests. Thank you for flagging this! I've since fixed!